server/datastore/psql/database.go

```diff
@@ -22,10 +21,14 @@ func (db *DB) Init(config *datastore.ServerConfig) (error) {
 		UseFallbackTLS:    false,
 		FallbackTLSConfig: nil,
 	}
-	conn, err := pgx.Connect(connConfig)
+	connection, err := pgx.Connect(connConfig)
 	if err != nil {
 		return err
 	}
+<<<<<<< HEAD
```

Comment:
Can you fix the merge conflicts? Just delete the lines you don't want

Reply:
fixed merge conflicts